demo/predictions/css/legend.css

```diff
@@ -0,0 +1,47 @@
+fieldset {
+    border: 1px solid#ddd;
+    padding: 5px 12px;
+}
+
+legend {
+    font-size: 18px;
+    padding: 0 10px;
+    margin: auto;
+}
+
+.state-predicted-on-time .legend li:first-child::before {
+    background-color: var(--color-state-predicted-on-time);
+}
+
+.state-predicted-late .legend li:first-child::before {
+    background-color: var(--color-state-predicted-late);
```

Comment:
**question:**  Why there is another variable with the same value (`--color-path-predicted-late`) ?
It's confusing and it's weird to add the 2 colors on the legend 🙁

Reply:
Same as for the other color, go to keep a single color